File: task-03/Javascript/Sub task 2/subtask2.js
const fs = require('fs');

const inputFilename = 'input.txt';
const outputFilename = 'output.txt';

// Read the content of input.txt
fs.readFile(inputFilename, 'utf8', (err, data) => {
    if (err) {
        console.error(`Error reading file: ${err.message}`);
        return;
    }

    // Write the content to output.txt
    fs.writeFile(outputFilename, data, (err) => {
        if (err) {
            console.error(`Error writing to file: ${err.message}`);
            return;
        }

        console.log(`Content successfully written to ${outputFilename}`);
    });
});

File: task-03/Elixir/Sub task 2/subtask2.exs
defmodule FileCopy do
  def copy_content do
    input_filename = "input.txt"
    output_filename = "output.txt"

    case File.read(input_filename) do
      {:ok, content} ->
        case File.write(output_filename, content) do
          :ok -> IO.puts("Content successfully written to #{output_filename}")
          {:error, reason} -> IO.puts("Error writing to file: #{reason}")
        end

      {:error, reason} ->
        IO.puts("Error reading file: #{reason}")
    end
  end
end

FileCopy.copy_content()

